vue/test/mocks/mixins/locale.js

```diff
@@ -0,0 +1,12 @@
+export const localeMockMixin = {
+    computed: {
+        locale() {
+            return (value, defaultValue) => value || defaultValue;
+        }
+    },
+    methods: {
+        locale(value, defaultValue) {

```

Comment:
@gcandal having locale method plus locale computed, works?

Reply:
It works but doesn't make sense. Fixed.